common/shlibs

```diff
@@ -96,6 +96,16 @@ libglib-2.0.so.0 glib-2.76.0_1
 libgmodule-2.0.so.0 glib-2.76.0_1
 libgio-2.0.so.0 glib-2.76.0_1
 libgobject-2.0.so.0 glib-2.76.0_1
+libgthread-2.0.so.0 glib-2.74.0_1
+libglib-2.0.so.0 glib-2.74.0_1
+libgmodule-2.0.so.0 glib-2.74.0_1
+libgio-2.0.so.0 glib-2.74.0_1
+libgobject-2.0.so.0 glib-2.74.0_1
```

Comment:
Addubg the glib shared libaries here again looks wrong.

Reply:
It is :( Git tricked me rebasing